svelte-kit-scss/src/lib/components/shared/layouts/BoxLayout.svelte

```diff
@@ -35,6 +40,9 @@
       border: $border;
       border-radius: 6px;
       background-color: variables.$white;
+      &.empty {
```

Comment:
This looks like a fix to a problem we shouldn't have. As your solution above demonstrates, we typically know when the layout is empty. In this case, we can prevent rendering the whole layout altogether.

Just my take glancing at this code. I might be missing things.